components/mpas-ocean/src/mode_forward/mpas_ocn_forward_mode.F

```diff
@@ -657,7 +662,12 @@ function ocn_forward_mode_run(domain) result(ierr)!{{{
 
       ! initialize time-varying forcing
       call ocn_time_varying_forcing_init(domain)
-      call ocn_time_varying_forcing_get(domain % streamManager, domain, domain % clock)
+
+      ! if not using RK4, calculate time varying forcing terms once per
+      ! time-step as opposed at each RK substage as implemented in RK4 
+      if (timeIntegratorChoice /= 4) then
```

Comment:
I wonder if now that we have the capability to compute the forcing correctly, we should allow the possibility of not doing it. For instance, instead of having a condition on the type of integrator here, we could have something like `config_update_forcing_at_intermediate_timesteps` that is also used in the RK4 timestepping code and sets all the `forcingTimeIncrementRK4` to zero if false. In this way, for coupled runs, one can decide to have a lighter coupling, but if one wants to do convergence tests and use RK4 as a reference solution to validate other time stepping schemes it is still possible turning that flag on.

Reply:
```suggestion
      if (timeIntegratorChoice /= timeIntRK4) then
```